fe/lib/components/QuestionnaireBuilder/app.stories.tsx

```diff
@@ -6,9 +6,9 @@ import { storiesOf } from 'sku/@storybook/react';
 
 import { QuestionnaireBuilder } from './QuestionnaireBuilder/QuestionnaireBuilder';
 
-storiesOf('App', module)
-  .add('seekAnz', () => <QuestionnaireBuilder />)
-  .addDecorator((story) => (
+storiesOf('QuestionnaireBuilder', module)
+  .add('Builder', () => <QuestionnaireBuilder />)
+  .addDecorator(story => (
```

Comment:
It seems that IDE integrations aren't picking up the Prettier config correctly 😢

I'll try to have a squiz at why later

Reply:
Yeah :/ I pushed the fix from running the `sku format`, was going to look into it also.